src/linker/implementation.py

```diff
@@ -1,4 +1,5 @@
 from pathlib import Path
+from re import M
```

Comment:
This doesn't appear to be used--I'm guessing from another PR?

Reply:
whoops